### PR TITLE
Remove unnecessary 2nd call to SwaggerParser.

### PR DESCRIPTION
**Pull request by pablograna**
Both CodeGenMojo and CodegenConfigurator where parsin the swagger input.
The result in CodeGenMojo was discarded. I simply removed the line in
CodeGenMojo.

### CODE REVIEW

**Comment by wing328** [comment]
Thanks for the PR but your commit (as shown in the Commits tab) is not linked to your Github account, which means this PR won't count as your contribution in https://github.com/swagger-api/swagger-codegen/graphs/contributors.

Let me know if you need help fixing it.

Ref: http://stackoverflow.com/questions/3042437/change-commit-author-at-one-specific-commit

**Comment by pablograna** [comment]
I think I've fixed it, now the page shows the commits.

File: modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/HaskellServantCodegen.java [hunk 1]
```diff
@@ -432,11 +431,11 @@ private String makeQueryListType(String type, String collectionFormat) {
             case "pipes": return "(QueryList 'PipeSeparated (" + type + "))";
             case "multi": return "(QueryList 'MultiParamArray (" + type + "))";
             default:
-                throw new NotImplementedException();
+                throw new UnsupportedOperationException();
```
Comment (wing328) [inline modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/HaskellServantCodegen.java]: I suggest we keep `NotImplementedException` as existing programs may be a dependency on this. If we switch it to `UnsupportedOperationException`, we'll need to flag this change as breaking change, which seems not necessary.

**Comment by wing328** [comment]
Yes, I think you've fixed it as confirmed in the commit tabs.

Btw, I would suggest you to create a new branch for the change as per Git best practice.

**Comment by pablograna** [comment]
Here is the 'clean' PR, just the line removed in a new branch:

https://github.com/swagger-api/swagger-codegen/pull/3005

Should I close this PR?

**Pull request closed without merging**